src/common/api.ts

```diff
@@ -21,3 +21,7 @@ export async function didKey(): Promise<string> {
 
   return didCache.didKey
 }
+
+export default {
+  didKey
+}
```

Comment:
Just out of curiosity, why are we doing this again?

Reply:
let's us do 
```ts
import { api } from '../common'
// or
import api from '../common/api'
//instead of 
import * as api from '../common/api'
```

Not necessary, I just personally like the semantics better. Also works better if we ever need to mock it (jest mocks only really seem to work at the module level)